applications/ContactStructuralMechanicsApplication/python_scripts/auxiliar_methods_solvers.py

```diff
@@ -0,0 +1,227 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+# Importing the Kratos Library
+import KratosMultiphysics as KM
+
+import KratosMultiphysics.StructuralMechanicsApplication as SMA
+import KratosMultiphysics.ContactStructuralMechanicsApplication as CSMA
+
+def print_on_rank_zero( *args):
+    # This function will be overridden in the trilinos-solvers
+    KM.Logger.PrintInfo(" ".join(map(str,args)))
+
+def print_warning_on_rank_zero( *args):
+    # This function will be overridden in the trilinos-solvers
+    KM.Logger.PrintWarning(" ".join(map(str,args)))
+
+def AuxiliarContactSettings():
```

Comment:
I would remove `Auxiliar` from all the methods in this file, I think it is duplicated
=> `auxiliar_methods_solver.AuxiliarContactSettings` => `auxiliar_methods_solver.ContactSettings`

btw also I think `auxiliar_solver_methods` sounds better, but its up to you

Reply:
I like auxiliar